src/pages/Beta/Pool/index.tsx

```diff
@@ -141,10 +155,23 @@ const PoolUI = () => {
   // add own v2
   if (ownminiChefStakingInfo.length > 0) {
     menuItems.push({
-      label: ownStakingInfoV1.length > 0 ? `${t('pool.yourFarms')} (V2)` : `${t('pool.yourFarms')}`,
+      label:
+        ownStakingInfoV1.length > 0 || ownPangoCheftStakingInfo.length > 0
```

Comment:
I didn't understand this condition here. why do we need `ownPangoCheftStakingInfo.length > 0` condition here ?